content/03.results.md

```diff
@@ -177,55 +174,44 @@ Since many cancers may have infiltrating immune cells, all immune cells were als
-The Jaccard similarity index is displayed in a heatmap, an example of which is shown in Figure {@fig:fig4}B.
+We calculate the Jaccard index between pairs of cell type labels to compare annotations between the two methods and display it in a heatmap (an example is shown in Figure {@fig:fig4}B).
+Jaccard index values close to 1 indicate high agreement and a high proportion of overlapping cells, which may indicate higher confidence predictions.
 
-The report also includes a diagnostic plot evaluating the confidence of cell type annotations determined by each method.
+The report also includes diagnostic plots for each method.
 To evaluate confidence in `SingleR` cell type annotations, the delta median statistic is calculated by subtracting the median score from the score associated with the assigned cell type label [@url:https://bioconductor.org/books/release/SingleRBook/annotation-diagnostics.html#based-on-the-deltas-across-cells].
-The distribution of delta median values for each cell type is shown in the cell type report, where a higher delta median statistic for a cell indicates higher confidence in the final cell type annotation.
-`CellAssign` calculates the probability that each cell belongs to each possible cell type provided in the reference, and the cell type label with the highest probability is assigned as the cell type for that cell.
-<!-- TODO: What exactly do we mean by reliable labels? Labels that are appropriate for the dataset -->
-These values range from 0 to 1, with larger values indicating greater confidence in a given cell type label, so we expect more confident labels to have most values close to 1.
-A plot displaying the distribution of all probabilities for each cell type is included in the cell type report.
+The cell type report shows the distribution of delta median values for each cell type. 
+A higher delta median statistic for a cell indicates higher confidence in the final cell type annotation.
+We also display the distribution of all probabilities calculated by `CellAssign`; more confident labels are expected to have many values close to 1.
 
 If the submitter provided cell types, the submitter annotations are compared to the annotations from both `SingleR` and `CellAssign`.
-A summary of this comparison is included in the cell type report along with a table summarizing the submitter cell type annotations and a UMAP plot where each cell is colored by the submitter annotation.
-The Jaccard similarity index is calculated for all pairs of cell type labels in submitter annotations and `SingleR` annotations and in submitter annotations and `CellAssign` annotations.
+The cell type report includes a summary of this comparison, a table summarizing the submitter cell type annotations, and a UMAP plot in which each cell is colored by the submitter annotation.
```

Comment:
The order of descriptions here seemed odd. So I tried combining all parts. It's probably too long now.
```suggestion
If the submitter provided cell type annotations, the cell type report also includes a table summarizing the submitter cell type annotations, a UMAP plot in which each cell is colored by the submitter annotation, and a comparison of the submitter annotations to the automated cell typing results from `SingleR` and `CellAssign`.
```